scripts/drevops/test.sh

```diff
@@ -75,8 +75,12 @@ if [ -z "${DREVOPS_TEST_TYPE##*unit*}" ]; then
   vendor/bin/phpunit "${phpunit_opts[@]}" docroot/modules/custom/ --filter '/.*Unit.*/' "$@" \
   || [ "${DREVOPS_TEST_UNIT_ALLOW_FAILURE}" -eq 1 ]
 
+  vendor/bin/phpunit "${phpunit_opts[@]}" docroot/themes/contrib/ --filter '/.*Unit.*/' "$@" \
```

Comment:
Added theme related unit tests - as i needed to add unit test for function I was changing.